### PR TITLE
fix `src/model/item.rs`

### PR DESCRIPTION
**Pull request by LelouchFR**
I got this Error:

```
thread 'tokio-runtime-worker' panicked at src/api/fetcher.rs:63:22:
called `Result::unwrap()` on an `Err` value: CookiesError(ReqWestError(reqwest::Error { kind: Decode, source: Error("missing field `is_for_swap`", line: 1, column: 15003) }))
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

to fix this issue, I did what @0xCAB0 said, which was changing https://github.com/ThalosES/vinted-rs/blob/main/src/model/item.rs#L131-L146 to be Option<_>

### CODE REVIEW

**Review by pxp9 — APPROVED** [review]
lgtm

**Review by 0xCAB0 — DISMISSED** [review]
Nice! Have you noticed any other parsing errors we might have missed?

**Comment by pxp9** [comment]
fix `src/model/item.rs`

**Review by pxp9 — CHANGES_REQUESTED** [review]
fix `src/model/item.rs`

**Comment by 0xCAB0** [comment]
@pxp9 meant to fix the file, not changing the PR name bahahahaha.
But it's alright

**Comment by LelouchFR** [comment]
> @pxp9 meant to fix the file, not changing the PR name bahahahaha. But it's alright

sure lol sorry, working on it :P

File: src/model/item.rs [hunk 1]
```diff
@@ -127,23 +127,56 @@ pub struct AdvancedItem {
...
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub is_closed: Option<i32>,
+}
+
+impl AdvancedItem {
+    fn display_option_bool(value: Option<bool>) -> String {
+        match value {
+            Some(true) => "true".to_string(),
+            Some(false) => "false".to_string(),
+            None => "None".to_string(),
```
Comment (0xCAB0) [inline src/model/item.rs:168]: Probably don't need the toString here. Also these kind of "utils" methods should not go in a class, the should be shared across the project, something like `src/lib/utils.rs/`

**Review by 0xCAB0 — CHANGES_REQUESTED** [review]
See comments

File: src/model/item.rs (hunk 1 above)
Comment (LelouchFR) [reply src/model/item.rs:168]: so should I create `src/lib/utils.rs` since there is none ?

**Comment by pxp9** [comment]
You should match tha value like this:

```rust

Some(val) => val.to_string()
None => "None".to_string()

```

File: src/model/item.rs (hunk 1 above)
Comment (0xCAB0) [reply src/model/item.rs:168]: Yeap

**Comment by LelouchFR** [comment]
normally it's now good and all running

**Review by 0xCAB0 — APPROVED** [review]
LGTM, nice job!

**Comment by LelouchFR** [comment]
> LGTM, nice job!

Thank you :) getting the hang of it now :D

**Pull request merged**